### DISCUSSION TITLE
[General Support]: Update Frigate LXC 0.14.0 to 0.14.1 not working

### DISCUSSION THREAD

**Discussion by xekil** [blakeblackshear/frigate#13427] [General Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Hello,

Updating from Frigate LXC 0.14.0 to [0.14.1](https://github.com/blakeblackshear/frigate/releases/tag/v0.14.1) (which has just been released) does not work.

Thanks.

```
    ______     _             __
   / ____/____(_)___ _____ _/ /____
  / /_  / ___/ / __ `/ __ `/ __/ _ \
 / __/ / /  / / /_/ / /_/ / /_/  __/
/_/   /_/  /_/\__, /\__,_/\__/\___/
             /____/

Loading...
 ✗ There is currently no update path available.
root@frigate:~# ^C
root@frigate:~# 

```

### Steps to reproduce

1. Launching the Frigate LXC script

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/frigate.sh)"`

### Version

0.14.0-

### In which browser(s) are you experiencing the issue with?

128.0.6613.114 (Build official) (64 bits)

### Operating system

Proxmox

### Install method

Script LXC

### Network connection

Wired

**Comment by hawkeye217** [comment, marked as answer]
Frigate does not manage the tteck Proxmox script. Please file a bug report with them instead.

**Reply by xekil** [reply]
Oops, sorry I was convinced I was posting on TTECK's Github
